src/views/Lottery/components/PastDrawsHistory/PastDrawsHistoryCard.tsx

```diff
@@ -1,18 +1,74 @@
-import React from 'react'
-import { Heading, Card, CardBody } from '@pancakeswap-libs/uikit'
+import React, { useState } from 'react'
+import { Heading, Card, CardBody, Text } from '@pancakeswap-libs/uikit'
 import useI18n from 'hooks/useI18n'
+import styled from 'styled-components'
+import Select, { OptionProps } from 'components/Select/Select'
 import HistoryChart from './HistoryChart'
 import Legend from './Legend'
 
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-direction: column-reverse;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+`
+
+const LabelWrapper = styled.div`
+  > ${Text} {
+    font-size: 12px;
+  }
+
+  margin-top: 24px;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    margin-top: 0;
+  }
+`
+
 const PastDrawsHistoryCard: React.FC = () => {
   const TranslateString = useI18n()
+  const [showLast, setShowLast] = useState<'max' | number>(50)
+  const handleShowLastChange = (option: OptionProps): void => {
+    setShowLast(option.value)
+  }
 
   return (
     <Card>
       <CardBody>
         <Heading size="md">{TranslateString(746, 'History')}</Heading>
-        <Legend />
-        <HistoryChart />
+        <Wrapper>
+          <Legend />
+          <LabelWrapper>
+            <Text>{TranslateString(999, 'SHOW LAST')}</Text>
```

Comment:
Please keep this normal case `Show Last` and capitalize it with the `textTransform` property on the `<Text />` component.

Reply:
done